Umee-Bots/lendingPool-reentrancy/src/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import { encodeFunctionSignature } from "forta-agent-tools/lib/utils";
+
+export interface AgentConfig {
+  lendingPoolAddress: string;
+  reentrancyBlacklist: string[];
+}
+
+const getSigHashes = (signaturesArray: string[]): string[] => {
+  return signaturesArray.map((signature) => {
+    return encodeFunctionSignature(signature);
+  });
+};
+
+const createFinding = (initialCallSelector: string, lendingPoolCallSelector: string): Finding => {
+  return Finding.fromObject({
+    name: "Detect Lending Pool Reentrancy",
+    description: "Detect when a reentrancy happened in the LendingPool contract",
```

Comment:
Suggestion: something like `"Reentrancy detected in call to LendingPool contract"`